main/glossary/index.md

```diff
@@ -602,6 +602,27 @@ Secure ECMAScript has been renamed to [Hardened JavaScript](#hardened-javascript
 
 An imaginary currency Agoric documentation uses in examples.
 
+## Smart Contract
+
+A smart contract is a self-executing contract with the terms of the agreement directly written into lines of code. These contracts are stored and executed on a blockchain network. Smart contracts automatically enforce and execute the terms of a contract when predefined conditions are met, eliminating the need for intermediaries and reducing the risk of human error or manipulation. Like user accounts on a chain, smart contracts can "own" or hold digital assets themselves.
```

Comment:
This is not the definition we use at Agoric.

Our definition is:

> A smart contract is a contract-like arrangement expressed in code, where the behavior of the program enforces the terms of the contract.

 - https://papers.agoric.com/smart-contracts-encyclopedia-entry/